resolwe_bio/docker_images/rnaseq/packages-manual/trimmomatic.sh

```diff
@@ -10,12 +10,19 @@ download_and_verify \
     http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/Trimmomatic-\${version}.zip \
     Trimmomatic-\${version}
 
-cat >trimmomatic.sh <<EOL
+cat >trimmomaticSE <<EOL
```

Comment:
The _here_ documents in Bash that I've seen usually use `EOF` as the limit string and reverse the arguments. Something like:

```
cat << EOF > /tmp/yourfilehere
These contents will be written to the file.
        This line is indented.
EOF
```